core-meetings/2019-11-oss-meeting.md

```diff
@@ -0,0 +1,33 @@
+# Open Source Maintenance: November 2018 Core Meeting
+
+As stewards of React Native engineers worldwide, we've contemplated ways to improve alignment and visibility of the framework's direction, where community members and Facebook alike joined to discuss issues and propose responses.
+
+The community team is made up of volunteers who help maintain React Native, contribute to education materials, and improve the React Native developer's environment.
+
+## The core discussion
+
+This time, many members from both the community and the FB team met to talk about the Open Source repository and its maintenance.
```

Comment:
We don't have to specifically call about FB team - they are also contributors, just like others in the community. There are a few other places in the document where this stands out.

Reply:
And it's intentional: they are not "just like others", the way FB uses RN internally is way different (as you also stated in your comments about releases below). The main reason for clearly stating that the FB team was participating to this meeting is so that it's transparent that you are involved, which also reassures developers about the commitment from FB in RN.